### DISCUSSION TITLE
Missing Icon on pi display

### DISCUSSION THREAD

**Discussion by skar-cgn** [photobooth-app/photobooth-app#125] [Q&A] [answered]
Hi all, 
I have the app running on a pi 4 with an attached display. 

On this display the icons (e.g. on the home screen) like cam icon or hearts are not working. Only two rectangles are visible. 
On a connection via notebook and firefox browser, the icons are normally visbile. 

Same for the cheese icon. 

I would asume a font or similar is missing. Any ideas ?

Regards
Ralf 
<img width="1202" alt="2023-11-11 19_44_45-fotobox (fotobox) – VNC Viewer" src="https://github.com/photobooth-app/photobooth-app/assets/56639989/9cbc3a86-38ba-40ab-8f3d-71d64b6b0296">

**Comment by mgineer85** [comment, marked as answer]
Hi! Yes, it seems you are missing a font. The font is usually installed before the photobooth app: https://photobooth-app.org/setup/installation/#install-system-dependencies

It's the following package:
sudo apt-get -y install fonts-noto-color-emoji

**Reply by skar-cgn** [reply]
Thanks a lot, works now as expected.
